components/paypal/actions/get_user_info/get_user_info.mjs

```diff
@@ -0,0 +1,23 @@
+import app from "../../app/paypal.app.mjs";
+
+export default {
+  key: "paypal-get-user-info",
+  name: "Get User Info",
+  description: "Shows user profile information. [See docs here](https://developer.paypal.com/docs/api/identity/v1)",
```

Comment:
```suggestion
  description: "Displays user profile information. [See docs here](https://developer.paypal.com/docs/api/identity/v1)"
```

Reply:
Changed 'Shows' to 'Displays' to follow Google's Developer docs style guide (Use present tense and active voice)